spec/services/ccms/requestors/case_add_requestor_xml_blocks/non_passported_variable_attributes_spec.rb

```diff
@@ -320,6 +321,28 @@ module Requestors # rubocop:disable Metrics/ModuleLength
           end
         end
 
+        context 'vehicle attributes' do
+          let(:vehicle) { legal_aid_application.vehicle }
+          context 'applicant has no vehicle' do
+            before { vehicle.update! estimated_value: nil }
+            let(:attrs) { %w[CARANDVEH_INPUT_B_14WP2_28A CARANDVEH_INPUT_C_14WP2_25A CARANDVEH_INPUT_C_14WP2_26A CARANDVEH_INPUT_D_14WP2_27] }
+            it 'does not generate the attributes' do
+              attrs.each do |attr_name|
+                block = XmlExtractor.call(xml, :vehicle_entity, attr_name)
+                expect(block).not_to be_present
+              end
+            end
+          end
+
+          context 'applicant has vehicle' do
```

Comment:
i think this test might need to be removed, there is a section missing, but again it is in master already